runtimes/huggingface/mlserver_huggingface/common.py

```diff
@@ -42,10 +45,12 @@ class Config:
     # for translation task, required a suffix to specify source and target
     # related issue: https://github.com/SeldonIO/MLServer/issues/947
     task_suffix: str = ""
+    auto_loader_name: Optional[str] = "AutoModel"
```

Comment:
Should we use a `PyObject` instead? You can check out the `Settings` class implementation for an example on how it could be used:

https://github.com/SeldonIO/MLServer/blob/27ca2573c740109e28da69690c8341260161b6aa/mlserver/settings.py#L90

Also, as a separate point, is `auto_loader_name` the best field name? Should it be just `model_loader`?